diff --git a/node_modules/rcinfo/lib/index.js b/node_modules/rcinfo/lib/index.js
index d82b4a1..d67400b 100644
--- a/node_modules/rcinfo/lib/index.js
+++ b/node_modules/rcinfo/lib/index.js
@@ -14,7 +14,7 @@
     if (process.platform !== "win32") {
       // replace executable with wine      
       args.unshift(executable); 
-      executable = "wine";
+      executable = ['arm64', 'x64'].includes(process.arch) ? 'wine64' : 'wine';
     }
 
     child = execFile(executable, args, options, function(error, stdout, stderr){
